bin.src/imsim.py

```diff
@@ -50,6 +50,10 @@
                     "This will be prepended to any existing IMSIM_IMAGE_PATH "
                     "environment variable, for which $CWD is included by "
                     "default.")
+parser.add_argument('--ckpt_archive_dir', type=str, default=None,
+                    help="Archive directory for checkpoint files. "
+                    "If None, then delete them (if the checkpointing.cleanup "
```

Comment:
Some users might not be aware of the config file and think this refers to a non-existent option.  Perhaps "if the checkpointing.cleanup variable in the config file being used is True"?